### PR TITLE
spherical harmonic gravity

### PR DESCRIPTION
**Pull request by mrtreasurer**
Added two Python files: gravity_spherical_harmonic, load_gravity_model
Added ten gravity models inside gravity_models

### CODE REVIEW

**Comment by darioizzo** [comment]
Great PR thanks! Super useful!! Before merging this we need:

 * [ ] Properly deal with the introduction of jit as a dependency (detect its installed and in case import the stuff)
* [ ] Write some unit tests for the imports and for the acc (maybe take from matlab some points and compare)
* [ ]

File: pykep/test.py [hunk 1]
```diff
@@ -212,6 +417,8 @@ def run_test_suite(level=0):
     suite.addTest(lambert_test_case())
     suite.addTest(mga_1dsm_test_case())
     suite.addTest(gym_test_case())
+    suite.addTest(spherical_harmonics_loader_test_case())
```
Comment (darioizzo) [inline pykep/test.py]: These tests need to be run only if numba is detected

File: pykep/util/load_gravity_model.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        s = row[3]
+
+        c_array[degree, order] = c
+        s_array[degree, order] = s
+
+    return c_array, s_array
+
+
+if __name__ == "__main__":
+    load_coefficients("C:/Users/Bert van den Abbeele/Documents/Python/HarmonicsPropagator/tests/jgl150q1.txt")
```
Comment (darioizzo) [inline pykep/util/load_gravity_model.py]: you may want to remove this or use a relative path

**Comment by darioizzo** [comment]
Theres some issue with the numba detection in extensions ... See the CI logs. This can only be merged if CI passes (green tick)

**Comment by mrtreasurer** [comment]
The problem was with the __init__.py.in file overwriting __init__.py, causing __extensions__ to not contain 'numba'

**Comment by mikeheddes** [comment]
I think because of line [23 of CMakeLists.txt](https://github.com/esa/pykep/blob/master/CMakeLists.txt#L23) that `__init__.py` should not be committed to the repo since CMake will generate the file at compile time from `__init__.py.in`. Or there is some particular reason for `__init__.py` to be in the repo that I don't know of.

I would suggest first making a commit in which you remove `__init__.py` then making a commit in which you add `pykep/__init__.py` to `.gitignore` so the file is removed and won't be accidentally added again in the future.

**Pull request merged**